istioctl/cmd/root.go

```diff
@@ -225,7 +225,6 @@ debug and diagnose their Istio mesh.
 	experimentalCmd.AddCommand(mesh.UninstallCmd(loggingOptions))
 	experimentalCmd.AddCommand(configCmd())
 	experimentalCmd.AddCommand(workloadCommands())
-	experimentalCmd.AddCommand(revisionCommand())
```

Comment:
do we normally/can we keep it in `x` as well, for backwards compat?

Reply:
makes sense, can't think of any drawbacks keeping it around (for a while)

Reply:
When a command graduates, we keep it in the old place but print a message to the user reminding them of the new place.

The implementation is one line, wrapping the command in `softGraduatedCommand()`.  Unfortunately that function became dead and was deleted.  See https://github.com/istio/istio/pull/26563/files .  Restore softGraduatedCommand() and wrap with it.

Reply:
Added back in, thanks.